tests/Geocoder/Tests/Provider/GoogleMapsTest.php

```diff
@@ -196,60 +196,61 @@ public function testGeocodeBoundsWithRealAddressForNonRooftopLocation()
     public function testGeocodeWithRealAddressReturnsMultipleResults()
     {
         $provider = new GoogleMaps($this->getAdapter());
-        $results  = $provider->geocode('Paris');
+        $results  = $provider->geocode('i');
```

Comment:
Ah, you get new caches.. Please rename the file "tests/.cached_responses/0e41628de6d90aa50c426e12391617abc3ccf5ff" to "tests/.cached_responses/65760b8fc8d216f997a2e5b2ce499881426f7353".

Now it will work with "Paris"